CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
 - Upgrade [`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) version from `v0.1.12` to `v0.2.0-beta.3`
 - Upgrade Kubernetes version from `kubernetes-1.13.4` to `kubernetes-1.14.1`
 - Upgrade `github.com/operator-framework/operator-lifecycle-manager` version from `b8a4faf68e36feb6d99a6aec623b405e587b17b1` to `0.10.1`
+- CRD file names were previously of the form `<group>_<version>_<kind>_crd.yaml`. Now that CRD manifest `spec.version` is deprecated in favor of `spec.versions`, i.e. multiple versions can be specified in one CRD, CRD file names have the form `<group>_<resource>_crd.yaml`. `<resource>` is the plural lower-case CRD Kind found at `spec.names.plural`.
```

Comment:
Changing from `<group>_<version>_<kind>_crd.yaml` to `<group>_<resource>_crd.yaml` isn't a breaking change?

Will any existing CLI commands like `csv-gen` or `generate openapi` now break if users have manifest in the old format `<group>_<version>_<kind>_crd.yaml`?

Reply:
It is indeed a breaking change for our testing infrastructure. `generate` commands will not break (`openapi` will write new CRD files), while `gen-csv` will only break if the CSV config contains specific file names and not just `deploy/crds`.

Regardless this should be moved to the "Breaking Changes". I will also move the dependency upgrades.

Reply:
@estroz Is this already updated to be in the breaking changes section?

Also is it possible to rebase this `refactor/controller-runtime-v0.2.0` branch from the master since this CHANGELOG is now for v0.9.0.

Or do you want to hold off for this PR and do it later?

Reply:
I haven't updated it yet, was waiting for more responses to comments. I'll update it now and rebase.